commcare/src/main/java/org/motechproject/commcare/pull/CommcareFormImporterImpl.java

```diff
@@ -194,6 +197,8 @@ private void importFormList(CommcareFormList formList) {
         for (CommcareForm form : Lists.reverse(formList.getObjects())) {
             FullFormEvent formEvent = new FullFormEvent(form.getForm(), form.getReceivedOn(), form.getConfigName());
 
+            lastStartedImportingFormId = form.getId();
```

Comment:
_lastImportedFormId_ contains last successfully imported form id, but on error we need to log which form was the last to be imported